packages/elements/src/pagination/index.ts

```diff
@@ -666,12 +506,18 @@ export class Pagination extends BasicElement {
         this.hasLastPage() && this.updateInputValue(this.internalMax);
         break;
       default:
-        return;
     }
 
-    if (event.key !== 'Tab') {
-      event.preventDefault();
+    /**
+     * Allow input keys (navigation, submit, number).
+     * Do not prevent default for `Tab` key, it should focus on next tabindex as the same native behaviour.
+     */
+    const allowKeys = ['ArrowLeft', 'Left', 'ArrowRight', 'Right', 'Backspace', 'Tab'];
```

Comment:
You do not need it as `input` event is covered

Reply:
Removed